packages/client/tests/e2e/enum-import-in-edge/package.json

```diff
@@ -4,8 +4,7 @@
   "main": "index.js",
   "scripts": {},
   "dependencies": {
-    "@prisma/client": "/tmp/prisma-client-0.0.0.tgz",
-    "wrangler": "3.109.3"
```

Comment:
This one should stay here. E2E tests are running in isolated Docker containers and aren't part of the workspace. Right now the test fails due to this:

```
$ pnpm wrangler dev --ip 127.0.0.1 --port 8787
undefined
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "wrangler" not found
```